schedule/yast/minimal+role_minimal.yaml

```diff
@@ -34,6 +34,19 @@ schedule:
   # Called on BACKEND: qemu, pvm_hmc
   - '{{grub_test}}'
   - installation/first_boot
+  - console/system_prepare
+  - console/consoletest_setup
+  - console/zypper_lr
+  - console/zypper_ref
+  - console/zypper_in
+  - console/firewall_enabled
+  - console/ncurses
+  - console/sshd_running
+  - console/sshd
```

Comment:
Second module does more magic and installs additional packages, and restarts `sshd` as a first step, so in case service is not running, we won't catch this.